.goreleaser.yml

```diff
@@ -82,6 +82,11 @@ dockers:
   dockerfile: 'docker/kanister-kubectl/Dockerfile'
   build_flag_templates:
   - "--build-arg=TOOLS_IMAGE=ghcr.io/kanisterio/kanister-tools:{{ .Tag }}"
+- ids:
+  - kando
+  image_templates:
+  - 'ghcr.io/kanisterio/redis-tools:{{ .Tag }}'
+  dockerfile: 'docker/redis-tools/Dockerfile'
```

Comment:
Let's not build the image in our CI.
We can add the steps to build the image and push it in the example README
```suggestion
```

Reply:
removed